src/System.Drawing.Common/src/System/Drawing/BufferedGraphicsContext.Unix.cs

```diff
@@ -28,62 +28,28 @@
 //
 //
 
-
 using System.ComponentModel;
 using System.Runtime.InteropServices;
 using System.Runtime.Serialization;
 
 namespace System.Drawing
 {
-    public sealed class BufferedGraphicsContext : IDisposable
+    public sealed partial class BufferedGraphicsContext : IDisposable
     {
-        private Size max_buffer;
-
-        public BufferedGraphicsContext()
-        {
-            max_buffer = Size.Empty;
-        }
```

Comment:
So this is more than just code consolidation, but actually fixing the behavior of the Unix implementation to match the Windows one?

Reply:
Essentially the code consolidation (in favour of Windows where applicable) fixes the behaviour, yes